docs/api/desktop-capturer.md

```diff
@@ -1,7 +1,10 @@
 # desktopCapturer
 
-> List `getUserMedia` sources for capturing audio, video, and images from a
-microphone, camera, or screen.
+> Provides information about media sources that can be used to capture audio and
```

Comment:
These API summaries are usually written in the imperative style. I think `Get` or `Access` would be preferable to `Provides` here.